src/configs/getAndroidConfig.ts

```diff
@@ -15,10 +15,16 @@ limitations under the License.
 */
 
 import { Platform } from "style-dictionary/types/Platform";
+import { TransformedToken } from "style-dictionary/types";
 import { Theme } from "../@types";
+import { isCoreColor, isNotCoreColor } from "../filters/isCoreColor";
 import _ from "lodash";
 
-export default function getAndroidConfig(theme: Theme): Platform {
+function isFontType(token: TransformedToken) {
+  return ['typography', 'fontFamilies', 'fontWeights', 'letterSpacing', 'fontSizes', 'lineHeights'].includes(token.type);
+}
+
+export function getAndroidConfig(theme: Theme): Platform {
   return {
     transforms: [
       "camelCaseDecimal",
```

Comment:
If we're going to keep that separation of configs, we'll need to remove a lot of transforms from the array below. And keep only the ones relevant to the output token name, and color transformations

Reply:
This has been optimized in fa0bcd2879715f3ae7873ac1746bf824a6000ad5 (although maybe some of them could still be removed).